docs/api/decorators/searchIndexes.md

```diff
@@ -0,0 +1,75 @@
+---
+id: searchIndexes
+title: '@searchIndex'
+---
+
+**Typings:**
+
+```ts
+function searchIndex<T extends BeAnObject = BeAnObject>(description: SearchIndexDescription): ClassDecorator
```

Comment:
```suggestion
function searchIndex(description: SearchIndexDescription): ClassDecorator
```

`T` is not used anywhere and the code version also does not have it.

even with this change, it does not really match the code definition